System/Posix/ARX/CLI/Options.hs

```diff
@@ -98,6 +101,8 @@ qPath                        =  tokCL QualifiedPath
 shared                      ::  ArgsParser Bool
 shared                       =  True <$ arg "--shared"
 
+tmpdir                       :: ArgsParser Path
+                             = arg "--tmpdir" >> QualifiedPath
```

Comment:
`=␣arg "--tmpdir" >> QualifiedPath` -> `=␣␣arg "--tmpdir" >> QualifiedPath` for consistency with surrounding code.